scapy/fields.py

```diff
@@ -1121,6 +1121,7 @@ def any2i(self, pkt, x):
     def i2repr(self, pkt, x):
         return _EnumField.i2repr(self, pkt, x)
 
+
```

Comment:
Could you please remove this useless change to preserve this file's code history? Thanks!

Reply:
as I touched that part of the code (by adding new fields but later on I moved them to another location)  I honored the boy scout rule and added a newline between the two classes to follow pep8. but I see your point - 3da7f3f